src/main/resources/swagger/api-docs.yaml

```diff
@@ -2704,6 +2704,44 @@ paths:
         500:
           description: Internal Error
 
```

Comment:
The Swagger update below is mostly copied from Rawls'. I'm not sure if that was the right thing to do. It has the `Model` and `Example Value` fields empty for `submissionsCount` on the Swagger page.